src/pages/Authentication/SignUp/SignUp.js

```diff
@@ -0,0 +1,84 @@
+import { useEffect } from "react"
+import { Link } from "react-router-dom"
+import "../Auth.css"
+
+export const SignUpPage = () => {
+    useEffect(()=>{
+        document.title = "Sign Up | BlueBasket"
+    })
+    return (
+        <div className="main-wrapper signup-page-content display-justify-center display-align-center">
+        <div className="image-container">
+            <img src="/assets/ecomm-login.png" alt="header-img"/>
+        </div>
+        <div className="card-container">
+            <div className="card for-login">
+                <div className="content">
+                    <h3>Sign Up</h3>
+                    <ul>
+                        <li>
+                            <label for="Email-Address" className="semi-bold">Email</label>
```

Comment:
instead of for, change it to forLabel as it will throw multiple warnings and fill your console